logging/unit_tests/test_client.py

```diff
@@ -92,11 +96,10 @@ def test_no_gax_ctor(self):
 
     def test_sinks_api_wo_gax(self):
         from google.cloud.logging._http import _SinksAPI
-        from google.cloud.logging import client as MUT
-        from google.cloud._testing import _Monkey
 
-        with _Monkey(MUT, _USE_GAX=False):
-            client = self._make_one(self.PROJECT, credentials=_Credentials())
+        client = self._make_one(
+            self.PROJECT, credentials=_Credentials(),
+            use_gax=False)
 
```

Comment:
[This][1] is a property test, not a constructor test.

The "environment" sniff [happens only in the constructor][3] and [there is already a test for that][2].

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/115748814bf6a59d99c13e070dfdcfbdec5e865d/logging/unit_tests/test_client.py#L93
[2]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/115748814bf6a59d99c13e070dfdcfbdec5e865d/logging/unit_tests/test_client.py#L80
[3]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/115748814bf6a59d99c13e070dfdcfbdec5e865d/logging/google/cloud/logging/client.py#L79-L82